cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +315,30 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@test_requirements.infra_hosts
+@pytest.mark.meta(coverage=[1738664])
```

Comment:
@prichard77 Is this test case going to be backported to 5.10? If so, you should list that BZ here. If not, you must use

```python,
@pytest.mark.meta(blockers=[BZ(1738664, forced_streams=["5.10"])], coverage=[1738664])
```

Reply:
Will this keep PRT from running this against 5.10? Do I need to leave "coverage" here, or update to automate? And do I replace line 321 with your suggestion or append your line?

Reply:
1. Yeah, this will cause PRT to skip the test case due to the blocking BZs. Also applies to any automation/jenkins runs. 
2. sorry, you should use `automates` here, I copypastad this line
3. Replace